apps/dav/lib/CalDAV/Schedule/IMipPlugin.php

```diff
@@ -199,6 +189,12 @@ public function schedule(Message $iTipMessage) {
 		// we also might not have an old event as this could be a new
 		// invitation, or a new recurrence exception
 		$attendee = $this->imipService->getCurrentAttendee($iTipMessage);
+		// Don't send emails to things
+		if($this->imipService->isRoomOrResource($attendee)) {
+			$this->logger->debug('No invitation sent as recipient is room or resource');
```

Comment:
```suggestion
			$this->logger->debug('No invitation sent as recipient is room or resource', [
			    'attendee' => $attendee->getValue(),
			]);
```

if we want to make this easier to debug in production

Reply:
Smort

Reply:
ALso this might blow up in our faces with email aliases btw. Will open a follow up ticket.